src/styles/_header.scss

```diff
@@ -22,7 +22,7 @@
 
   &__item {
     position: relative;
-    color: #060b35;
+    color: #{$main-color-header};
```

Comment:
don't need tipe #{} when connect variables in one file through import, use only $ (everywhere)